<?xml version="1.0"?>
  <!-- Sample based on https://gist.github.com/jbohren/4be3e7cf96fa25c36188 -->
  <launch>

  <arg name="video_path" default="$(find image_pub)/data" />
  <arg name="ground_video_path" default="$(find image_pub)/data" />
  <arg name="camera_info_url" default="file:///$(find image_pub)/param/camera.yaml"/>
  <arg name="camera_info_topic" default="/camera/camera_info"/>
  <arg name="image_topic" default="/camera_front_image_raw"/>
  <arg name="ground_video_topic" default="/camera_ground_image_raw"/>
  <arg name="pub_rate" default="24.0"/>
  <arg name="start_sec" default="0"/>
  <arg name="repeat" default="true"/>

  <node name="mp4" pkg="image_pub" type="image_pub_node" output="screen">
    <param name="video_path" value="$(arg video_path)"/>
    <param name="frame_id" value="simulation_camera" />
    <param name="camera_topic" value="$(arg image_topic)" />
    <param name="camera_info_url" value="$(arg camera_info_url)" />
    <param name="camera_info_topic" value="$(arg camera_info_topic)" />
    <param name="pub_rate" value="$(arg pub_rate)" />
    <param name="start_sec" value="$(arg start_sec)" /> 
    <param name="repeat" value="$(arg repeat)" /> 
  </node>

  <node name="ground_mp4" pkg="image_pub" type="image_pub_node" output="screen">
    <param name="video_path" value="$(arg ground_video_path)"/>
    <param name="frame_id" value="simulation_ground_camera" />
    <param name="camera_topic" value="$(arg ground_video_topic)" />
    <param name="camera_info_url" value="$(arg camera_info_url)" />
    <param name="camera_info_topic" value="$(arg camera_info_topic)" />
    <param name="pub_rate" value="$(arg pub_rate)" />
    <param name="start_sec" value="$(arg start_sec)" /> 
    <param name="repeat" value="$(arg repeat)" /> 
  </node>

</launch>